docs/91_version_history.rst

```diff
@@ -131,6 +99,42 @@ Release Date: 2019-08-15
        - The GUI was not correctly interpreting the value of the readonly attribute of configuration items when the value was anything other than true. This has been resolved.
        - The FogLAMP GUI RPM package had an error that caused installation to fail on some systems, this is now resolved.
 
+- **Plugins**
+
+    - New Features:
+
+       - A new filter has been created that looks for changes in values and only sends full rate data around the time of those changes. At other times the filter can be configured to send reduced rate averages of the data.
+       - A new south plugin has been created to read the data files produced by a Sarcos Robotics exoskeleton.
+       - A new rule plugin has been implemented that will create notifications if the value of a data point moves more than a defined percentage from the average for that data point. A moving average for each data point is calculated by the plugin, this may be a simple average or an exponential moving average.
+       - A new south plugin has been created that supports the DNP3 protocol.
+       - A south plugin has been created based on the Google TensorFlow people detection model. It uses a live feed from a video camera and returns data regarding the number of people detected and the position within the frame.
+       - A south plugin based on the Google TensorFlow demo model for people recognition has been created. The plugin reads an image from a file and returns the people co-ordinates of the people it detects within the image.
+       - A south plugin based not the Google TensorFlow demo model for object detection has been creates.
+       - A new north plugin has been added that creates an OPC UA server based on the data ingested by the FogLAMP instance.
+       - A new south plugin has been created that implements a DNP3 master.
```

Comment:
we have above ` - A new south plugin has been created that supports the DNP3 protocol.`